### PR TITLE
Update image url to come from field in graphql query

### PR DESCRIPTION
**Pull request by connorreinhold**
## Overview

Instead of manually creating an imageurl, use the field that the backend passes to us in graphql. Fixed bug for Louie's image and Anabel's image not showing up. Addresses issue #131 

## Changes Made

Changed getImageUrl method of EateryBaseModel and added imageUrl to the AllEateriesQuery

## Test Coverage

Simulator testing - images showed up

</details>

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import android.text.Spannable;
 import android.text.SpannableString;
 import android.text.style.StyleSpan;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java]: remove

File: app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java [hunk 2]
```diff
@@ -17,6 +18,7 @@
 import com.cornellappdev.android.eatery.model.DiningHallModel;
 import com.cornellappdev.android.eatery.model.EateryBaseModel;
 import com.cornellappdev.android.eatery.model.enums.PaymentMethod;
+import com.cornellappdev.android.eatery.network.NetworkUtilities;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java]: might be wrong, but we don't need this import?

File: app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java [hunk 3]
```diff
@@ -1,6 +1,9 @@
 package com.cornellappdev.android.eatery.network;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java]: do we need these imports?

File: app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java (hunk 2 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java]: deleted

File: app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java (hunk 3 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java]: Deleted unused ones

File: app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java (hunk 2 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java]: deleted

File: app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java (hunk 3 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java]: deleted unused ones - most we need tho

**Pull request merged**